### ISSUE TITLE
Missing VERSION file in pip installation

### ISSUE THREAD

**Issue by mihaic** [twolfson/restructuredtext-lint#33]
Installing using `pip` produces a broken installation because `VERSION` is not installed:
```
$ pip install --user restructuredtext_lint
Collecting restructuredtext_lint
Requirement already satisfied (use --upgrade to upgrade): docutils<1.0,>=0.11 in ./.local/lib/python3.4/site-packages (from restructuredtext_lint)
Installing collected packages: restructuredtext-lint
Successfully installed restructuredtext-lint-0.15.0
$ rst-lint 
Traceback (most recent call last):
  File "/home/mihai/.local/bin/rst-lint", line 7, in <module>
    from restructuredtext_lint.cli import main
  File "/home/mihai/.local/lib/python3.4/site-packages/restructuredtext_lint/cli.py", line 11, in <module>
    with open(os.path.join(os.path.dirname(__file__), 'VERSION'), 'r') as version_file:
FileNotFoundError: [Errno 2] No such file or directory: '/home/mihai/.local/lib/python3.4/site-packages/restructuredtext_lint/VERSION'
```

You should include `VERSION` in `package_data` in `setup.py`:
https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files

**Comment by andrew-tipton-cgg** [comment]
This issue just bit us this morning.  It appears that 0.15.0 was released overnight, and as a result our unit tests are now failing.  We're going to pin the version to 0.14.3 for now, but it was a slightly unpleasant experience.

**Comment by twolfson** [comment]
Ah, sorry folks. I don't have time to follow @mihaic's advice (due to wanting to manually verify rather than release blindly) but I do have time for a revert (which has no manual testing)

**Comment by twolfson** [comment]
This should be reverted and back to working in `0.16.0`.

**Comment by twolfson** [comment]
We have completed a patched version and released it in `0.17.0`. I previously reproduced with the steps listed and can no longer so I think we are good. Thanks for the quick report and help everyone =)
